src/ray/core_worker/task_interface.cc

```diff
@@ -54,9 +54,24 @@ Status CoreWorkerTaskInterface::CreateActor(
   std::vector<ObjectID> return_ids;
   return_ids.push_back(ObjectID::ForTaskReturn(task_id, 1));
   ActorID actor_creation_id = ActorID::FromBinary(return_ids[0].Binary());
+  ray::ActorDefinitionDescriptor actor_definition_descriptor;
+  switch (function.language) {
+  case WorkerLanguage::PYTHON:
+    RAY_CHECK(function.function_descriptor.size() == 3);
+    actor_definition_descriptor.push_back(function.function_descriptor[0]);
+    actor_definition_descriptor.push_back(function.function_descriptor[1]);
+    break;
+  case WorkerLanguage::JAVA:
+    RAY_CHECK(function.function_descriptor.size() == 3);
+    actor_definition_descriptor.push_back(function.function_descriptor[0]);
```

Comment:
I think it's okay to just save actor creation task's function descriptor in the handle. Then we don't need to introduce this new concept `ActorDefinitionDescriptor`

Reply:
Changed to `actor_creation_task_function_descriptor`